Headlines/src/Services/ReactionsService.swift

```diff
@@ -9,7 +9,6 @@
 import UIKit
 import CloudKit
 
-// swiftlint:disable force_try
 class ReactionsService: HTTPService {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/00fb2a4e33be33a18e690ab5eb06c7befa08ef9a/Headlines/src/Services/ReactionsService.swift#L12"></span>SwiftLint rule ‘force_try’ did not trigger a violation in the disabled region. Please remove the disable command.<br />
<code>superfluous_disable_command</code> <code>ReactionsService.swift:12</code></td>
    </tr>
  </tbody>
</table>